lightning/src/routing/router.rs

```diff
@@ -165,14 +165,22 @@ pub struct RouteParameters {
 
 	/// The CLTV on the final hop of the failed payment path.
 	pub final_cltv_expiry_delta: u32,
+
+	/// The maximum total CLTV delta we accept for the route.
+	pub max_total_cltv_expiry_delta: u32,
 }
 
 impl_writeable_tlv_based!(RouteParameters, {
 	(0, payee, required),
 	(2, final_value_msat, required),
 	(4, final_cltv_expiry_delta, required),
+	(6, max_total_cltv_expiry_delta, required),
 });
 
+/// Maximum total CTLV difference we allow for a full payment path
+pub const DEFAULT_MAX_TOTAL_CLTV_DELTA: u32 = 1000;
```

Comment:
`DEFAULT_MAX_TOTAL_CLTV_EXPIRY_DELTA`?

Reply:
Yeah, I hesitated to go there since that is one long variable name. But you're right, for consistencies sake it's likely better. Done in https://github.com/lightningdevkit/rust-lightning/pull/1234/commits/b7cb0f3eca8f491de09c97e41b8f090c3d27264e.